### DISCUSSION TITLE
Apex DNS not redirecting to WWW as suggested

### DISCUSSION THREAD

**Discussion by ikehCharles** [vercel/community#6359] [Help]
### Summary

After setting up the A record on my apex domain and a CNAME on sub-domain WWW, when I access www(subdomain) I see the application but when I try accessing through apex domain, It keeps loading without a redirect.

### Example

https://geekgrin.com

### Steps to Reproduce

Try accessing https://geekgrin.com and it wont redirect to www.geekgrin.com

**Comment by amyegan** [comment]
Hey @ikehCharles. Are you still seeing this problem? I visited the site and didn't have any problem with the `www` redirect just now.

**Reply by ikehCharles** [reply]
Still experiencing same problem, This is what I am getting
<img width="1419" alt="Screenshot 2024-04-01 at 7 36 45 AM" src="https://github.com/vercel/community/assets/67435200/3f662737-5ba7-4c27-81e9-79d2fa9e3a40">
